kaos/algorithm/coord_conversion.py

```diff
@@ -0,0 +1,90 @@
+from math import sqrt,sin,cos,pi,atan,tan
+from numpy import rad2deg,deg2rad
+from astropy import coordinates, units
+from astropy.time import Time
+from kaos.algorithm import Vector3D
+
+def lla_to_ecef(lat, lon, alt=0):
+    """converts latitude, longitude, and altitude to earth-centered, earth-fixed (ECEF) Cartesian.
+
+    Args:
+    lat = geodetic latitude (decimal degrees)
+    lon = longitude (decimal degrees)
+    alt = height above WGS84 ellipsoid (m)
+
+    Returns:
+    (x,y,z) such that:
+      x = ECEF X-coordinate (m)
+      y = ECEF Y-coordinate (m)
+      z = ECEF Z-coordinate (m)
+
+    Notes: This function assumes the WGS84 model. Latitude is customary geodetic (not geocentric).
+
+    Source: Adapted from a Matlab script by Michael Kleder
+    https://www.mathworks.com/matlabcentral/fileexchange/7942-covert-lat-lon-alt-to-ecef-cartesian
+
+    based on: Department of Defense World Geodetic System 1984"
+              Page 4-4
+              National Imagery and Mapping Agency
+              Last updated June, 2004
+              NIMA TR8350.2
+    """
+
+    # WGS84 ellipsoid constants:
+    a = 6378137
+    e = 8.1819190842622e-2
+    #decimal degrees to radians
+    lat = lat/360*2*pi
```

Comment:
Why are you doing the conversion yourself, you have functions to this for you :p